### PR TITLE
Conformance tests for pool_test and containerdisk

### PR DESCRIPTION
**Pull request by fossedihelm**
### What this PR does
This PR aims to identify and label the tests in the pool_test and containerdisk files.

### Release note

```release-note
NONE
```

### CODE REVIEW

File: tests/pool_test.go [hunk 1]
```diff
@@ -157,7 +157,7 @@ var _ = Describe("[sig-compute]VirtualMachinePool", decorators.SigCompute, func(
 		return createVirtualMachinePool(newPoolFromVMI(libvmifact.NewCirros()))
 	}
 
-	DescribeTable("pool should scale", func(startScale int, stopScale int) {
+	DescribeTable("pool should scale", decorators.Conformance, func(startScale int, stopScale int) {
```
Comment (xpivarc) [inline tests/pool_test.go:160]: Unfortunately this is not GA API, please drop this commit (the first one is still welcome)

File: tests/container_disk_test.go [hunk 2]
```diff
@@ -54,13 +54,12 @@ import (
...
 
 	var virtClient kubecli.KubevirtClient
-	var err error
 
 	BeforeEach(func() {
 		virtClient = kubevirt.Client()
 	})
 
-	DescribeTable("should", func(image string, policy k8sv1.PullPolicy, expectedPolicy k8sv1.PullPolicy) {
+	DescribeTable("should", decorators.Conformance, func(image string, policy k8sv1.PullPolicy, expectedPolicy k8sv1.PullPolicy) {
```
Comment (xpivarc) [inline tests/container_disk_test.go:62]: Please see https://github.com/kubevirt/kubevirt/pull/10441/files#diff-005747350099de38fcd22401693fe634b2c4607b4deec581b298efe440b93d2f

**Comment by fossedihelm** [comment]
/hold
waiting for https://github.com/kubevirt/kubevirt/pull/10441

File: tests/container_disk_test.go [hunk 3]
```diff
@@ -142,7 +142,7 @@ var _ = Describe("[rfe_id:588][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 
 	Describe("[rfe_id:273][crit:medium][vendor:cnv-qe@redhat.com][level:component]Starting with virtio-win", func() {
 		Context("with virtio-win as secondary disk", func() {
-			It("[test_id:1467]should boot and have the virtio as sata CDROM", func() {
+			It("[test_id:1467]should boot and have the virtio as sata CDROM", decorators.Conformance, func() {
```
Comment (xpivarc) [inline tests/container_disk_test.go:145]: I think this is just a variation, what bus I can ask for... I would not include it.

**Review by xpivarc — COMMENTED** [review]
/approve
/hold

File: tests/container_disk_test.go (hunk 3 above)
Comment (fossedihelm) [reply tests/container_disk_test.go]: Sure! Done

**Comment by fossedihelm** [comment]
@xpivarc @0xFelix 2 approve and 0 lgtm :)

**Review by xpivarc — APPROVED** [review]

**Pull request merged**